content/applications/hr/payroll.rst

```diff
@@ -159,8 +159,10 @@ Reporting section
    :align: center
    :alt: New work entry type form with all fields to be filled in.
 
-Working schedules
------------------
+.. _payroll/working-times:
+
+Working times
```

Comment:
Same here, the accepted change included the custom anchor AND the 15.0 heading which differs from the 16.0 heading. 
```suggestion
Working schedules
-----------------
```

Reply:
Thanks for catching that- I still don't quite understand why the forward porting didn't include this as well.